ERCS/eip-draft_security_tokens_gesact.md

```diff
@@ -0,0 +1,271 @@
+---
```

Comment:
```suggestion
---
number: 7551
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Many thanks, @abcoathup!